<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticipantDialog</class>
 <widget class="QDialog" name="ParticipantDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="18" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QComboBox" name="TCPcomboBox">
     <item>
      <property name="text">
       <string>TCP LAN Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP WAN Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="IntraprocesscheckBox">
     <property name="text">
      <string>Intraprocess</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QSpinBox" name="spin_domainId">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="value">
      <number>80</number>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="serverIP_label">
     <property name="text">
      <string>Server IP:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="listeningPort_label">
     <property name="text">
      <string>Listen Port:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="serverPort_label">
     <property name="text">
      <string>Server Port:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLineEdit" name="lineEdit_server_ip"/>
   </item>
   <item row="30" column="3">
    <widget class="QPushButton" name="pushButton_start">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="30" column="4">
    <widget class="QPushButton" name="pushButton_stop">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Transport</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="UDPcheckBox">
     <property name="text">
      <string>UDP</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QCheckBox" name="statisticsCheckBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Only available when Fast DDS is compiled with Statistics Module.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Active Statistics</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QCheckBox" name="monitorServiceCheckBox">
     <property name="text">
      <string>Active Monitor Service</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Advanced options</string>
     </property>
    </widget>
   </item>
   <item row="25" column="1">
    <widget class="QCheckBox" name="typeinformationCheckBox">
     <property name="text">
      <string>Autofill type information</string>
     </property>
    </widget>
   </item>
   <item row="26" column="1">
    <widget class="QCheckBox" name="lossCheckBox">
     <property name="text">
      <string>Enable UDP Sample Loss</string>
     </property>
    </widget>
   </item>
   <item row="26" column="3">
    <widget class="QSpinBox" name="lossSpin">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="26" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="27" column="0">
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ROS</string>
     </property>
    </widget>
   </item>
   <item row="28" column="1">
    <widget class="QCheckBox" name="rosTopicCheckBox">
     <property name="text">
      <string>Use ROS 2 Topics</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="TCPcheckBox">
     <property name="text">
      <string>TCP</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="SHMcheckBox">
     <property name="text">
      <string>Shared Memory</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Same host delivery</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QSpinBox" name="spin_listen_port">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Domain</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QSpinBox" name="spin_server_port">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
